sys/include/net/gnrc/ipv6/nib/conf.h

```diff
@@ -138,6 +138,17 @@ extern "C" {
 #endif
 #endif
 
+/**
+ * @brief   Handle route information option in neighbor advertisements
+ *          This is a non-standard extension proposed by
+ *          draft-templin-6man-rio-redirect-08
```

Comment:
We can use hyper refs so lets use them ;-).

```suggestion
 *          [draft-templin-6man-rio-redirect-08]
 *          (https://tools.ietf.org/html/draft-templin-6man-rio-redirect-08)
```